src/util-memcmp.c

```diff
@@ -35,8 +35,8 @@
 
 static int MemcmpTest01 (void)
 {
-    uint8_t a[] = "abcd";
-    uint8_t b[] = "abcd";
+    uint8_t a[] = "abcd            ";
+    uint8_t b[] = "abcd            ";
```

Comment:
Why do we need all tests to have lengths of 16 ?

Reply:
Values less than the SIMD width fall back to the simple scaler code and don’t test the more complex parts of the implementation.

Reply:
Agree, we shouldn't update existing tests for this, but instead add a new set of tests.